scalding-parquet/src/main/scala/com/twitter/scalding/parquet/tuple/macros/Macros.scala

```diff
@@ -0,0 +1,9 @@
+package com.twitter.scalding.parquet.tuple.macros
+
+import scala.language.experimental.macros
+
+import com.twitter.scalding.parquet.tuple.macros.impl.SchemaProviderImpl
+
+object Macros {
+  def caseClassParquetSchema[T]: _root_.parquet.schema.MessageType = macro SchemaProviderImpl.toParquetSchemaImp[T]
```

Comment:
Hi @ianoc , you are right. There's no need for macro implicits. I keep macros and delete these macro implicits in my latest commits.